app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="your-tasks">
+  <h2>YOUR TASKS</h2>
+  <ol>
+    <% @tasks.each do |task| %>
+      <li>
+        <% if task.complete == true %>
+          <strong id="complete"><%=task.title%></strong>
```

Comment:
We should use a class here instead of an ID for `"complete"`, because this code is inside of a loop. That means we will potentially get multiple `strong` tags on the page, all with the same ID.